account.cmd/src/main/java/com/ultimatesoftware/banking/account/cmd/service/controllers/AccountController.java

```diff
@@ -18,41 +21,41 @@
     private CommandGateway commandGateway;
 
     @PostMapping("accounts")
-    public CreateAccountCommand addAccount(@Valid @RequestBody AccountCreationDto account) {
+    public ResponseEntity<String> addAccount(@Valid @RequestBody AccountCreationDto account) {
```

Comment:
afaik you dont need to explicitly return ResponseEntity inside the RestController, unless you need to change the HTTP status code or set specific headers, or might return different types.

Reply:
True, we could just leave it void, but it feels weird since all the others do.